components/Chips/src/MDCChipView.m

```diff
@@ -423,6 +440,8 @@ - (void)setSelected:(BOOL)selected {
 #pragma mark - Layout
 
 - (void)layoutSubviews {
+  [super layoutSubviews];
```

Comment:
May want to consider pulling this out to a separate PR in case we need to revert this PR.

Reply:
Done.